src/main/java/com/ixi_U/common/exception/enums/UserException.java

```diff
@@ -0,0 +1,22 @@
+package com.ixi_U.common.exception.enums;
+
+import com.ixi_U.common.exception.BaseException;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+
+@Getter
+@RequiredArgsConstructor
+public enum UserException implements BaseException {
+
+    USER_NOT_FOUND(HttpStatus.NOT_FOUND, "사용자를 찾을 수 없습니다.");
+
+    private final HttpStatus httpStatus;
+    private final String message;
+
+    @Override
+    public String getExceptionName() {
+
+        return this.name();
+    }
+}
```

Comment:
```suggestion
}

```
코딩 컨벤션에 따라 파일 끝에는 항상 빈 라인으로 마무리해주세요